src/grep.rs

```diff
@@ -96,13 +99,15 @@ fn grep_bytes(
 
   let mut matched = false;
 
+  let buffer = buffer.as_ref();
```

Comment:
Could you move this to the beginning of the function, just like in `grep_filename`? Just to keep things the same way.

Reply:
Sorry, I've just done it.